hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -44,44 +46,69 @@ function ActionLabelSettings() {
   return <>Action Label Settings coming here</>;
 }
 
-function WebhookActionerRender() {
-  return <Card>This is a card displaying a WebhookActionerRender</Card>;
+function WebhookActionerRender({url, headers, webhookType}) {
+  return (
+    <Card>
+      <h5>WebhookActionerRender</h5>
```

Comment:
Use `<CardHeader>`, `<CardBody>` and `<CardFooter>`. And if you have a set of attributes, like URL, Webhook Type and Headers, you probably should put a table inside the `<CardBody>`.

Reply:
https://react-bootstrap.github.io/components/cards